deploy/vagrant/conf/tachyon.yml

```diff
@@ -17,3 +17,10 @@ Release:
 #  for virtualbox, machine memory is configured in conf/init.yml:Memory
 #  for aws, machine memory is decided by instance type
 WorkerMemory: 1GB
+
+# Number of Tachyon masters in the cluster, should be at least 1.
+# The cluster will be in fault tolerant mode if the number is larger than 1.
+# When it is in fault tolerant mode, the masters will be named as TachyonMaster, TachyonMaster1, TachyonMaster2, etc,
```

Comment:
That's fine, I just wanted to keep the previous convention, ok, I'll update to 1, 2, 3 and the documentation.